### ISSUE TITLE
Add watering forms to website

### ISSUE THREAD

**Issue by jodinewman** [ForetCapitaleForest/actions#399]

**Comment by jodinewman** [comment]
[Watering forms](https://foretcapitaleforest.ca/water-event-reporting/) have been uploaded for each site. It is currently set to send email reminders to my email only, so that someone can test out the forms. I can put in the emails for each site once we confirm they are how we want them. 

I also made a new, public google calendar specifically for watering events that the forms update.

**Comment by smlambert** [comment]
thanks @jodinewman - great progress!

when people are testing these out, try loading them from a phone to see what they look like (since we envision that is how some folks will report in)

also, remember that 'user experience' is key (ease of use, less clicks required), so consider not make people scroll down to fill in a form, does it work to have the form at the top of the page, with the extra info at the bottom?  once someone sees this page one or two times, do they refer to the maps and text for subsequent visits, do we need a 'select all' option for what was watered?


Noting that under a different issue I am going to be trying to reduce the menu bloat we have at the top of the page, down to 3 menu items (which then will have sub-items):
- About
  - Our Team
  - Jobs
  - Contact us
  - Projects
  - Resources
- Get Involved
  - Volunteer
    - Submit a Watering report
  - Donate
  - Partner
- News
  - Blog
  - In the news (links from other news outlets about our work)

**Comment by jodinewman** [comment]
@smlambert I believe the user experience should be improved now - I have added accordion menus for the watering instructions/health and safety info so that you don't have to scroll through them all if you don't need to use it. I also added a select all option. 

Once you've reviewed, I can change it so the notification is sent to the respective Google group emails, unless there are any further comments!
